synth.py

```diff
@@ -5,46 +5,50 @@
 import os
 
 logging.basicConfig(level=logging.DEBUG)
-
-gapic = gcp.GAPICGenerator()
-v2_library = gapic.node_library(
+version='v2'
+gapic = gcp.GAPICMicrogenerator()
+v2_library = gapic.typescript_library(
   'bigtable',
   'v2',
-  config_path='/google/bigtable/artman_bigtable.yaml'
-)
+  generator_args={
+    "grpc-service-config": f"google/bigtable/{version}/bigtable_grpc_service_config.json",
+    "package-name": f"@google-cloud/bigtable",
+    "main-service": f"bigtable"
+    },
+    proto_path=f'/google/bigtable/{version}',
+    extra_proto_files=['google/cloud/common_resources.proto'],
+  )
 
 s.copy(
   v2_library,
-  excludes=['package.json', 'README.md', 'src/index.js', 'src/v2/index.js']
+  excludes=['package.json', 'README.md', 'src/index.ts', 'src/v2/index.ts', 'tsconfig.json', 'tslint.json']
```

Comment:
If we truly need to ignore `index.ts`, `tsconfig`, etc, we need comment blocks explaining why please. 

Also, I don't want to ignore `tsconfig.json` and `tslint.json`

Reply:
`bigtable` is a little bit special, that it combines `admin/v2` and `v2` client files into `src/v2`. So there's manual work in `src/v2/index.ts` and `src/index.ts` to add AdminClients.

And for `tsconfig.json` & `tslint.json` it has enabled some setting like ` "noImplicitAny": false` to make sure the manual layer `src/*.ts` pass the lint check. 
I can surely remove these files from `excluded = []` and manually modify json files in synth.py, but that may increase many more lines in synth.py. Thank you!

Reply:
Oh gross! 🙈  Skipping these for now is fine.  Can you please make sure there's a followup bug in the repo to fix the types and disable this though?  Maybe add a comment saying as much in this doc, referencing the issue?

Reply:
yes, sure! I added some comments in synth.py, and I will also create bugs for them for better tracking. Thanks for the suggestions!